src/Repository/CoordinateRepository.php

```diff
@@ -17,5 +17,17 @@ class CoordinateRepository extends ServiceEntityRepository
     public function __construct(RegistryInterface $registry)
     {
         parent::__construct($registry, Coordinate::class);
+
+    }
+
+    public function getCoordinates($bottomLeftLat, $bottomLeftLng, $topRightLat, $topRightLng){
+
+        $query = "SELECT id, name, address, latitude, longitude
```

Comment:
pamegink query parasyti parametrus paduodant per `bindValue` (https://www.doctrine-project.org/projects/doctrine-dbal/en/2.7/reference/data-retrieval-and-manipulation.html#binding-types)
tokiu budu apsisaugosim nuo sql injection